pvlib/spectrum.py

```diff
@@ -0,0 +1,318 @@
+r"""
+The ``spectrum`` module contains functions that implement models for the solar
+irradiance spectrum.
+"""
+
+import pvlib
+from pvlib.tools import cosd
+import numpy as np
+
+# SPECTRL2 extraterrestrial spectrum and atmospheric absorption coefficients
+_SPECTRL2_COEFFS = np.zeros(122, dtype=np.dtype([
```

Comment:
Any thoughts about putting these into a csv/txt file?

Reply:
Minor preference from me to keep them close to the code, but I can't deny that it's 70 lines of junk that don't really need to be here.  Happy to dump to a csv if people don't want it in the code.

Reply:
Unless we see a use for these values independent of the SPECTRL2 function I'd prefer to keep the "junk" in the source code.